.github/workflows/windows.yml

```diff
@@ -281,13 +281,13 @@ jobs:
 
       - name: Build cpp samples
         run: |
-          & ${{ env.SAMPLES_INSTALL_DIR }}/cpp/build_samples_msvc.bat -i ${{ env.INSTALL_DIR }} -b ${{ env.BUILD_DIR }}/cpp_samples
+          & ${{ env.SAMPLES_INSTALL_DIR }}/cpp/build_samples_msvc.ps1 -InstallDirectory ${{ env.INSTALL_DIR }} -BuildDirectory ${{ env.BUILD_DIR }}/cpp_samples
         env:
           CMAKE_COMPILE_WARNING_AS_ERROR: 'ON'
 
       - name: Build c samples
         run: |
-          & ${{ env.SAMPLES_INSTALL_DIR }}/c/build_samples_msvc.bat -i ${{ env.INSTALL_DIR }} -b ${{ env.BUILD_DIR }}/c_samples
+          & ${{ env.SAMPLES_INSTALL_DIR }}/c/build_samples_msvc.ps1 -InstallDirectory ${{ env.INSTALL_DIR }} -BuildDirectory ${{ env.BUILD_DIR }}/c_samples
```

Comment:
You can keep the old line. That way you will test both bat and ps1. Just leave a comment, that two build calls are intentional